### ISSUE TITLE
Poor oral mucosal wound healing

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11680]
# Poor oral mucosal wound healing

## Proposed definition

An impairment of the normal tissue-repair process following injury to the oral mucosa, resulting in delayed or incomplete wound closure.

## Parent term

- `HP:0031815 Abnormal oral physiology`

## Synonyms

- Delayed oral mucosal healing
- Impaired oral wound healing

## Associated diseases

_None provided_

## Contributor

- ORCID: 0009-0001-4998-8617

**Comment by pnrobinson** [comment]
@evanc1648

Adding new term: 
- ID: HP:6001524
- label: Poor oral mucosal wound healing
- parents: [Abnormal oral physiology (HP:0031815)](https://hpo.jax.org/app/browse/term/HP:0031815)
- pmid: PMID:34439831
- def: An impairment of the normal tissue-repair process following injury to the oral mucosa, resulting in delayed or incomplete wound closure.
- comment: In general, wounds in the oral mucosa heal relatively fast and with little scarring, despite being continuously exposed to motion, tension, mechanical abrasion, and a great variety of microbes.
- orcid: orcid:0009-0001-4998-8617
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11680
- synonyms:
